reactive/kubernetes_worker.py

```diff
@@ -242,6 +242,35 @@ def shutdown():
     service_stop('snap.kube-proxy.daemon')
 
 
+@when_not('config.changed.ports-to-open')
+def open_close_ports_if_needed():
+    # 1) Check which ports are open and save to already_opened
+    # opened_ports return format: "icmp" or "80/tcp"
+    already_opened = set([p.split("/")[0] for p in hookenv.opened_ports()])
+    # 2) iterate over the list from ports-to-open and check if not present to alrady_opened
+    to_open = hookenv.config().get("ports-to-open")
+    if to_open is None:
+        to_open = ""
+    # Basic clean-up: remove any dups with set()
+    # clean-up string, breaking on , and replacing empty spaces
+    to_open_set = set([p.replace(" ","") for p in to_open.split(",")])
```

Comment:
wouldn't this turn the config "8   0,    4     4  3" into a valid config? Is that desired? It could be, but I'm just not sure the intent and why replace instead of trim.

Reply:
```python
to_open_set = set(p.strip() for p in to_open.split(','))
```